pages/email.tsx

```diff
@@ -1,18 +1,128 @@
+import { useFormik } from 'formik'
+import * as Yup from 'yup'
 import { GetStaticProps } from 'next'
 import { useTranslation } from 'next-i18next'
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
 import Layout from '../components/Layout'
+import { EmailEsrfRequestBody } from './api/email-esrf'
+import { EmailEsrf } from '../lib/EmailEsrfHook'
+import { useMemo } from 'react'
+import ErrorSummary, {
+  ErrorSummaryItem,
+  GetErrorSummary,
+} from '../components/ErrorSummary'
+import InputField from '../components/InputField'
+import ActionButton from '../components/ActionButton'
 
 export default function Email() {
   const { t } = useTranslation('email')
 
+  const initialValues: EmailEsrfRequestBody = {
+    dateOfBirth: '',
+    email: '',
+    firstName: '',
+    lastName: '',
+  }
+  const submittedKey = 'submitted'
+
+  const formik = useFormik<EmailEsrfRequestBody>({
+    initialValues,
+    validationSchema: Yup.object({
+      firstName: Yup.string().required('firstName.error.required'),
+      lastName: Yup.string().required('lastName.error.required'),
+      dateOfBirth: Yup.date()
+        .required('dateOfBirth.error.required')
+        .max(new Date(), 'dateOfBirth.error.current'),
+      email: Yup.string()
+        .required('email.error.required')
+        .email('email.error.valid'),
+    }),
+    validateOnBlur: false,
+    validateOnChange: false,
+    validateOnMount: false,
+    onSubmit: async (_, formikHelper) => {
```

Comment:
Do we want to add a cancel or reset button the for in case user wants to go back for any reason?

Reply:
we might... I was just focused on the form itself; can be added in when needed.